### PR TITLE
fix/logfile: remove old logfile on startup

### PR DESCRIPTION
**Pull request by Fraser999**
This is a temporary workaround for a bug in log4rs whereby the contents of a previous logfile aren't
cleared before writing to the file in a new run.

---
This change is [Reviewable](https://reviewable.io/reviews/maidsafe/safe_vault/520)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @maqi

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/bin/safe_vault.rs [hunk 1]
```diff
@@ -77,9 +78,12 @@ struct Args {
...
+    // TODO - remove the following line once maidsafe_utilities is updated to use log4rs v4.
+    let _ = fs::remove_file("Node.log");
+
     let args: Args = Docopt::new(USAGE)
-                         .and_then(|docopt| docopt.decode())
-                         .unwrap_or_else(|error| error.exit());
+        .and_then(|docopt| docopt.decode())
+        .unwrap_or_else(|error| error.exit());
 
     let name = config_file_handler::exe_file_stem().unwrap_or_else(|_| OsString::new());
```
Comment (dirvine) [inline src/bin/safe_vault.rs]: I wonder should the config file handler be using `create(true)` to provide this functionality (know it's not part of this PR though). If we want read or create then that would be the way I think.
